.gitignore

```diff
@@ -3,3 +3,13 @@
 /env/
 /paml.egg-info/
 /notebooks/.ipynb_checkpoints/*
+**/__pycache__
+notebooks/cluster*
+notebooks/*md
+pypaml.egg-info*
+notebooks/out
+notebooks/.*
+test/.*
+.vscode
+**/.DS_Store
+test/testfiles/igem_ludox_time_test.nt
```

Comment:
This shouldn't be in a .gitignore --- if there is a file getting emitted here, then it means the test should be getting run in a tmpdir instead.

This might mean the whole collection of notebook tests needs to involve copying into a tmpdir for execution during the test.

Reply:
Removed.  This file is the test rubric, so it shouldn't be ignored. It is written by the test (accidentaly uncommented) to update the rubric.